tests/ui/pattern/complexity_limit.rs

```diff
@@ -0,0 +1,106 @@
+#![feature(rustc_attrs)]
+#![pattern_complexity = "60"]
```

Comment:
Could you set this much higher, closer maybe to what rust-analyzer might set as default limit? Maybe `10000`, as long as the test still runs fast

Reply:
Done.